packages/govern-create/tasks/govern.ts

```diff
@@ -0,0 +1,87 @@
+import { ethers } from 'ethers'
+import { task } from 'hardhat/config'
+import {
+  uniqueNamesGenerator,
+  adjectives,
+  colors,
+  animals,
+} from 'unique-names-generator'
+
+function buildName(name: string | null): string {
+  const uniqueName =
+    name ??
+    uniqueNamesGenerator({
+      dictionaries: [adjectives, colors, animals],
+      length: 2,
+      separator: '-',
+    })
+
+  return process.env.CD ? `github-${uniqueName}` : uniqueName
+}
+
+const format = (address: string, name: string, network: string): string =>
+  `- ${name}: https://${
+    network === 'mainnet' ? '' : `${network}.`
+  }etherscan.io/address/${address}`
+
+task('deploy-govern', 'Deploys a Govern instance')
+  .addOptionalParam('factory', 'GovernBaseFactory address')
+  .addOptionalParam('useProxies', 'Whether to deploy govern with proxies')
+  .addOptionalParam('name', 'DAO name (must be unique at GovernRegistry level)')
+  .setAction(
+    async (
```

Comment:
We could have this also as a non-anonymous function and with a good name as the others. Would increase the readability.

Reply:
This is the usual way to write hardhat tasks. The name of the task is already stated (`deploy-govern`). Here we are just setting the action to be done. Why you consider it's not readable.